inc/html.class.php

```diff
@@ -3777,8 +3782,11 @@ static function showToolTip($content, $options=array()) {
     *    - user    : integer / restrict to a defined user (default -1 : no restriction)
     *    - option  : string / options to add to text field
     *    - display : boolean / if false get string
+    *    - type    : string / html5 field type (number, date, text, ...) defaults to 'text'
+    *    - required: boolean / whether the field is required
+    *    - rand    : integer / pre-exsting random value
     *
-    * @return nothing (print out an HTML div)
+    * @return void (print out an HTML div)
```

Comment:
depending on display option, the html can be returned (the comment was already bad previously)